blocks/callout/callout.css

```diff
@@ -0,0 +1,96 @@
+.callout {
```

Comment:
We should aim to support views down to 350px. Currently, the CTA buttons are hidden in smaller viewports.